universum/modules/launcher.py

```diff
@@ -358,7 +358,7 @@ def process_project_configs(self) -> configuration_support.Configuration:
         sys.path.append(os.path.join(os.path.dirname(config_path)))
 
         try:
-            with open(config_path) as config_file:
+            with open(config_path, encoding="utf-8") as config_file:
```

Comment:
Is there documented way to specify the encoding for .py files?
If yes, we should consider using it instead of hardcoding.

Reply:
Actually, PEP263 still works in Python 3. The only change is the default encoding. 

Please add the requirement for utf8 for configs to documentation.